library/alloc/src/string.rs

```diff
@@ -2196,6 +2196,9 @@ pub trait ToString {
 /// since `fmt::Write for String` never returns an error itself.
 #[stable(feature = "rust1", since = "1.0.0")]
 impl<T: fmt::Display + ?Sized> ToString for T {
+    // The general guide line is to not inline generic functions. However,
```

Comment:
s/guide line/guideline/

Is that really a guideline? I'm sure I've seen many generic functions marked with `inline`.

Reply:
The guideline is documented here: https://forge.rust-lang.org/libs/maintaining-std.html#is-that-inline-right .
Or I may infer it wrong.